2022q4/status.adoc

```diff
@@ -0,0 +1,57 @@
+
+Here are the reasons for such a change:
+
+* having two indipendent repositories requires to spend some time to make sure that both are in sync, which is being done manually. See for example commits such as
+link:https://github.com/freebsd/freebsd-quarterly/commit/4b8255e604dd0513e841aa8f3dce7741e78b999c[https://github.com/freebsd/freebsd-quarterly/commit/4b8255e604dd0513e841aa8f3dce7741e78b999c], which are not immediately clear in their commit messages about what is being done unless more time is invested to copy commit messages properly;
+
+* the FreeSD doc/ repository is self-hosted, while the status repository is hosted on GitHub. Since the contents of the self-hosted repository are mirrored, nothing is being lost in visibility with the repository merging.
+
+Some inconsitencies about the name of the team have also been found: the team has been refered as "quarterly", "quarterly status team", "status", "status team", "monthly" etc.
+So this issue is also being addressed.
+
+Please note that we are still working on these changes and that they might no be completed within the next quarter.
+The status team will take care to keep all information about report submissions up to date so that you always know how to submit your reports.
+
+==== Team naming
+
+Since "quarterly" might refers to quarterly reports but also to quarterly branches, using "quarterly" only could cause some confusion in some contexts.
+"quarterly-status" is likely a bad idea as well, as the frequency of reports publication might need to change in the future.
+Thus just "status" has been chosen: this is correct as quarterly status reports contain information about the status of the development of FreeBSD, it is frequency agnostic and it is consistent with link:https://www.freebsd.org/status/[its FreeBSD website section].
+
+The following link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=267813[e-mail addresses have been created]:
+
+* the main contact address for the status report team is now <status@FreeBSD.org>. Mails sent to <quarterly@FreeBSD.org> will still reach the team, but you are encouraged to use the new address;
+* the e-mail address for the status report submissions is now <status-submissions@FreeBSD.org>. Mails sent to <quarterly-submissions@FreeBSD.org> will still reach the team, but you are encouraged to use the new address;
+* the <quarterly-calls@FreeBSD.org> mailing list has been renamed to <status-calls@FreeBSD.org>.
+
+==== Report submissions
+
+Three different ways to submit reports will be provided:
+
+* submitting a review on Phabricator.
+  link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=267812[Two new Phabricator groups should be created]: "status-team" and "status-reviewers".
+  The first one will contain the official members of the status teams, while the second one will be open to join by anyone willing to review status reports and should be add by reports submitters as reviewers when they create their submissions.
```

Comment:
s/status teams/status team/ (unless I missed something and there's more than one)